recipes/liboptv/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: optv
+  version: {{ version }} 
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/{{ version }}.tar.gz
+  sha256: 4cafc70d966f2694cc936973eb2581d35bdffd6ec4b30f9ff99a1515c07f4042
+
+build:
+  number: 0
+  skip: True  # [not py3k]
```

Looking at the failures I think this was intended to be?
```suggestion
  skip: True  # [py3k]
```